verification/policy_test.go

```diff
@@ -0,0 +1,198 @@
+package verification
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func dummyPolicyStatement() (policyStatement TrustPolicy) {
+	policyStatement = TrustPolicy{
+		Name:                  "test-statement-name",
+		RegistryScopes:        []string{"test-registry-scope"},
+		SignatureVerification: "strict",
+		TrustStore:            "ca:test-store",
+		TrustedIdentities:     []string{"test-identity"},
+	}
+	return
+}
+
+func dummyPolicyDocument() (policyDoc PolicyDocument) {
+	policyDoc = PolicyDocument{
+		Version:       "1.0",
+		TrustPolicies: []TrustPolicy{dummyPolicyStatement()},
+	}
+	return
+}
+
+func writeToTempFile(policyDoc PolicyDocument) string {
+	tmpFile, _ := ioutil.TempFile(os.TempDir(), "notation-test-")
```

Comment:
nit: If you use [testing.T#TempDir](https://pkg.go.dev/testing#T.TempDir) instead of `os.TempDir` Go will make sure the temp directory is removed, so you won't have to call `os.Remove` later on.